WordPress/src/main/java/org/wordpress/android/ui/media/MediaSettingsActivity.java

```diff
@@ -651,6 +651,11 @@ private void setupImageSizeSeekBar() {
 
         int imageSizeKey = Arrays.asList(mImageSizeKeyArray).indexOf(mEditorImageMetaData.getSize());
 
+        //if the image size is something we can't recognize, default to the full size
+        if (imageSizeKey == -1) {
+            imageSizeKey = mImageSizeLabelArray.length - 1;
```

Comment:
I wonder what the unrecognized value may be. What do you think about adding some logging for it here? Thanks!

Reply:
Unrecognized value can really be anything - we are taking user input here. Having something that matches parameter regex (like `size-xl`) in the class attribute of an image will crash the media settings. 

Loggin is an excellent idea! Do you know if we have any helper class or tool for logging this kind of things (and retrieving them later?)?

Reply:
I see. It's not obvious that this is practically a user input field so, perhaps we can update the comment (Line 654) to include that bit of info.

About logging, I don't think we have a facility like that so, we can just use `AppLog` and hope to pick up the message via Helpshift when/if a user reports a problem.